monoio-http-client/examples/h2_tls.rs

```diff
@@ -0,0 +1,70 @@
+use std::time::Instant;
+
+use http::{request::Builder, Method};
+use monoio_http::common::body::{Body, FixedBody, HttpBody};
+
+#[monoio::main( enable_timer = true)]
+async fn main() {
+    let h2_client = monoio_http_client::Builder::new()
+        .http2_client()
+        .http2_max_concurrent_streams(200)
+        .build();
+
+    let base_url = "https://api.binance.com";
```

Comment:
I suggest using some neutral service like httpbin. I'm not sure if there are some policy issues here.